sourced/ml/algorithms/swivel.py

```diff
@@ -59,47 +59,52 @@
 import threading
 
 import numpy as np
-import tensorflow as tf
-from tensorflow.python.client import device_lib
-
-flags = tf.app.flags
-
-flags.DEFINE_string("input_base_path", None,
-                    "Directory containing input shards, vocabularies, "
-                    "and marginals.")
-flags.DEFINE_string("output_base_path", None,
-                    "Path where to write the trained embeddings.")
-flags.DEFINE_integer("embedding_size", 300, "Size of the embeddings")
-flags.DEFINE_boolean("trainable_bias", False, "Biases are trainable")
-flags.DEFINE_integer("submatrix_rows", 4096,
-                     "Rows in each training submatrix. This must match "
-                     "the training data.")
-flags.DEFINE_integer("submatrix_cols", 4096,
-                     "Rows in each training submatrix. This must match "
-                     "the training data.")
-flags.DEFINE_float("loss_multiplier", 1.0 / 4096,
-                   "constant multiplier on loss.")
-flags.DEFINE_float("confidence_exponent", 0.5,
-                   "Exponent for l2 confidence function")
-flags.DEFINE_float("confidence_scale", 0.25,
-                   "Scale for l2 confidence function")
-flags.DEFINE_float("confidence_base", 0.1, "Base for l2 confidence function")
-flags.DEFINE_float("learning_rate", 1.0, "Initial learning rate")
-flags.DEFINE_string("optimizer", "Adagrad",
-                    "SGD optimizer (tf.train.*Optimizer)")
-flags.DEFINE_integer("num_concurrent_steps", 2,
-                     "Number of threads to train with")
-flags.DEFINE_integer("num_readers", 4,
-                     "Number of threads to read the input data and feed it")
-flags.DEFINE_float("num_epochs", 40, "Number epochs to train for")
-flags.DEFINE_float("per_process_gpu_memory_fraction", 0,
-                   "Fraction of GPU memory to use, 0 means allow_growth")
-flags.DEFINE_integer("num_gpus", 0,
-                     "Number of GPUs to use, 0 means all available")
-flags.DEFINE_string("logs", "",
-                    "Path for TensorBoard logs (empty value disables them)")
-
-FLAGS = flags.FLAGS
+try:
```

Comment:
We cannot do this because we take this file from https://github.com/src-d/tensorflow-swivel

Wrap it on the upper level